src/tokens/TokenBalances.tsx

```diff
@@ -49,19 +51,23 @@ function TokenBalancesScreen({ navigation }: Props) {
   const walletAddress = useSelector(walletAddressSelector)
   const insets = useSafeAreaInsets()
 
+  const totalPositionsBalanceUsd = useSelector(totalPositionsBalanceUsdSelector)
+  const totalPositionsBalanceLocal = useDollarsToLocalAmount(totalPositionsBalanceUsd)
```

Comment:
any benefit to selecting the positions balances in usd and converting to local currency with this hook? the tokens selector uses local currency, should we also just return the local currency in the positions selector too?

Reply:
I'm gonna use this selector in other places and USD seemed the better.
My thinking was that the display in local currency should be handled by a component dedicated to this.
The tokens balance would eventually be refactored to return USD instead.